micrometer-observation/src/main/java/io/micrometer/observation/Observation.java

```diff
@@ -48,6 +50,8 @@
  */
 public interface Observation extends ObservationView {
 
+    InternalLogger logger = InternalLoggerFactory.getInstance(Observation.class);
```

Comment:
The `logger` is exposed as `public`. It seems to be an accidental leak.

Reply:
Darn, it is, let me fix it